hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/ConsumerProcess.java

```diff
@@ -26,35 +28,36 @@
 
     private final Retransmitter retransmitter;
 
-    private final java.util.function.Consumer<SubscriptionName> shutdownCallback;
+    private final java.util.function.Consumer<Signal> shutdownCallback;
 
-    private volatile boolean running = true;
+    private boolean running = true;
```

Comment:
I am pretty sure that this volatile here is unnecessary but after your suggestion I decided to revert this change as "future-proof".